runtime/v1/shim/service_linux.go

```diff
@@ -49,9 +49,11 @@ func (p *linuxPlatform) CopyConsole(ctx context.Context, console console.Console
 		cwg.Add(1)
```

Comment:
The stdin fifo will be open twice in shim. stdin copy goroutine exit, when fifo is broken or closed. 

So, I prefer to remove the `cwg.Done()` and `cwg.Add(1)` here.

Reply:
However, I can't reproduce it in my local 😂

Reply:
sorry for the wrong comment.